packages/mobile/android/app/proguard-rules.pro

```diff
@@ -35,6 +35,7 @@
 -keepclassmembers class * extends com.sun.jna.* { public *; }
 
 # Hermes
+-keep class com.facebook.hermes.unicode.** { *; }
```

Comment:
I'm not sure why this is needed since Hermes was already enabled.

Reply:
Was mentioned here: https://reactnative.dev/docs/0.66/hermes.